server/routes/index.js

```diff
@@ -1,376 +1,16 @@
-    appDb
-      .collection('users')
-      .findOne(
-        { uid: req.params.uid },
-        { _id: 0, blocked: 1 },
-        function (err, data) {
-          if (err) {
-            console.log(err)
-            return res.status(502).send(null)
-          } else if (!data || Object.keys(data).length === 0) {
-            return res.status(404).send(null)
-          } else {
-            return res.status(200).json(data['blocked'])
-          }
-        }
-      )
-  })
-  .post(ensureAdmin, function (req, res) {
-    if (Object.prototype.hasOwnProperty.call(req.body, 'blocked')) {
-      const { appDb } = req.app.locals
-      appDb
-        .collection('users')
-        .findOneAndUpdate(
-          { uid: req.params.uid },
-          { $set: { blocked: req.body.blocked } },
-          { returnOriginal: false },
-          function (err) {
-            if (err) {
-              console.log(err)
-              return res.status(502).send({ message: 'fail' })
-            } else {
-              return res.status(201).send({ message: 'success' })
-            }
-          }
-        )
-    } else {
-      return res.status(502).send({ message: 'fail' })
-    }
-  })
-
-router
-  .route('/api/v1/users/:uid/studies')
-  .get(ensureAdmin, function (req, res) {
-    const { appDb } = req.app.locals
-    appDb
-      .collection('users')
-      .findOne({ uid: req.params.uid }, { _id: 0, access: 1 }, function (err) {
-        if (err) {
-          console.log(err)
-          return res.status(502).send({ message: 'fail' })
-        } else {
-          return res.status(201).send({ message: 'success' })
-        }
-      })
-  })
-  .post(ensureAdmin, function (req, res) {
-    if (Object.prototype.hasOwnProperty.call(req.body, 'acl')) {
-      const { appDb } = req.app.locals
-      appDb
-        .collection('users')
-        .findOneAndUpdate(
-          { uid: req.params.uid },
-          { $set: { access: req.body.acl } },
-          { returnOriginal: false },
-          function (err) {
-            if (err) {
-              console.log(err)
-              return res.status(502).send({ message: 'fail' })
-            } else {
-              return res.status(201).send({ message: 'success' })
-            }
-          }
-        )
-    } else {
-      return res.status(502).send({ message: 'fail' })
-    }
-  })
-
-router
-  .route('/api/v1/users/:uid/configs/:config_id')
-  .get(ensureUser, function (req, res) {
-    const { appDb } = req.app.locals
-    appDb
-      .collection('configs')
-      .findOne(
-        { readers: req.params.uid, _id: new ObjectId(req.params.config_id) },
-        function (err, data) {
-          if (err) {
-            console.log(err)
-            return res.status(502).send({})
-          } else if (!data || Object.keys(data).length === 0) {
-            return res.status(404).send({})
-          } else {
-            return res.status(200).json(data)
-          }
-        }
-      )
-  })
-
-router
```

Comment:
We don't need this route? I thought we were using this admin/preferences stuff, unless we have duplicate code in `controllers`.

Reply:
The users controller and configurations controller handle all of that, the refactors were individual slices of work so there were multiple pages using these routes, over time we switched them over to the controller routes that have tests, etc... 

https://github.com/AMP-SCZ/dpdash/blob/1754680d5fc06453b356aecea215bc653227874b/server/routes/configurations.js#L16